location_register/converter/koatuu.py

```diff
@@ -1,148 +1,238 @@
+                object_citydistrict_name = self.get_lowercase_words_before_virgule(
+                    object_koatuu[self.OBJECT_NAME])
                 object_level_four = object_koatuu[self.LEVEL_FOUR]
-                self.save_to_city_or_citydistrict_table(data, object_level_four, object_citydistrict_name, district_items_list, district_dict, city_dict)
-                self.save_to_city_or_citydistrict_table(data, object_level_four, object_citydistrict_name, city_items_list, city_dict, citydistrict_dict)
-        print("Koatuu values saved")
-       
-    #writing entry to koatuu field in region table
-    def save_to_region_table(self, data, object_koatuu, region_dict, object_region_name):
-        if  not object_region_name in region_dict:
-            return
-        region_koatuu = region_dict[object_region_name]
-        region_koatuu.koatuu = object_koatuu[self.LEVEL_ONE]
-        region_koatuu.save(update_fields=['koatuu'])
-            
-    #writing entry to koatuu field in district table and level_two records of city_table
-    def save_to_district_table(self, data, object_koatuu, district_dict, city_dict, object_region_name, region_dict, district_items_list, KOATUU_VALUE):
-        for object_district_name in district_items_list:
-            if (object_district_name in district_dict) & (KOATUU_VALUE[2]!='1'):
-                district_koatuu = district_dict[object_district_name]
-                district_koatuu.koatuu = object_koatuu[self.LEVEL_TWO]
-                district_koatuu.save(update_fields=['koatuu'])
-            elif (object_district_name in city_dict) & (KOATUU_VALUE[2]=='1'):
-                city_koatuu = city_dict[object_district_name]
-                city_koatuu.koatuu = object_koatuu[self.LEVEL_TWO]
-                city_koatuu.save(update_fields=['koatuu'])
+                category_level_four = self.get_category_id(object_koatuu[self.CATEGORY])
+                self.save_to_city_or_citydistrict_table(
+                    object_level_four,
+                    object_citydistrict_name,
+                    district_items_list,
+                    district_dict,
+                    city_dict,
+                    category_level_four)
+                self.save_to_city_or_citydistrict_table(
+                    object_level_four,
+                    object_citydistrict_name,
+                    city_items_list,
+                    city_dict,
+                    citydistrict_dict,
+                    category_level_four)
+        print("Koatuu values to city and citydistrict saved")
 
-    #writing entry to koatuu field in city and citydistrict table
-    def save_to_city_or_citydistrict_table(self, data, object_level_number, object_level_name, level_items_list, up_level_dict, level_dict):
+    # writing entry to koatuu field in city and citydistrict table
+    def save_to_city_or_citydistrict_table(self, object_level_number, object_level_name, \
```

Comment:
redundant line break orerator
```suggestion
    def save_to_city_or_citydistrict_table(self, object_level_number, object_level_name,
```